libs/checkbox/src/lib/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <label>
+    <input
+      type="checkbox"
+      [disabled]="isDisabled"
+      [checked]="isChecked"
```

Comment:
You should use two-way binding to automatically have updated value in your property to then pass it to the output.